Tests/ClrAssembly/ClrAssembly.csproj

```diff
@@ -24,7 +24,7 @@
     <Csc TargetType="Library" References="@(ReferencePath)" Sources="%(LoadOrderCSFile.Identity)" OutputAssembly="$(OutputPath)\$(Prefix)%(LoadOrderCSFile.Filename).dll" />
   </Target>
 
-  <Target Name="ReachType" Inputs="@(ReachTypeInput)" Outputs="@(ReachTypeOutput)" DependsOnTargets="GetFrameworkPath">
+  <Target Name="ReachType" Inputs="@(ReachTypeInput)" Outputs="@(ReachTypeOutput)" DependsOnTargets="GetFrameworkPath" Condition="'$(MSBuildRuntimeType)' != 'Core'">
```

Comment:
Is this actually part of this change for removing those files?

Reply:
It's the only change needed to get it building with `dotnet build Dlr.sln` so I threw it in there.

Reply:
:+1: